app/views/styleguide/_menu.html.erb

```diff
@@ -31,4 +31,7 @@
   <li class="styleguide-nav__item">
     <a href="/styleguide/latestnews" class="styleguide-nav__item-link">Latest News</a>
   </li>
+  <li class="styleguide-nav__item">
+    <a href="/styleguide/downloads" class="styleguide-nav__item-link">Download Box</a>
+  </li>
 </ul>
```

Comment:
Same comment as below.

Reply:
Not sure how to do this in ruby, not a ruby developer

Reply:
I think of you as a jack of all trades. I can push something.

Reply:
@johnplayer1982 wdyt?

Reply:
Your solution in the recent commit is excellent, much nicer